doc/user/structure.rst

```diff
@@ -62,10 +62,17 @@ target
     heterogeneous dishes. The :samp:`{band}` is the single-letter abbreviation
     for the receiver band (``l``, ``s``, ``u`` or ``x``).
 
+    Additionally, for each cohort there is a target
+    :samp:`cohort/{cohort}/{band}`. For MeerKAT the cohort name is simply
+    ``meerkat``. For the MeerKAT Extension it will be ``meerkat_extension``.
+
     Note that even if ``individual`` is requested, many or all of the antennas
     may still share the same model if per-antenna models have not been
     produced.
 
+    The ``current/`` directory must contain all the targets defined
+    above, but the ``config/`` directory need not.
```

Comment:
I like! For me, the config is a concrete thing, so all indirection / mapping should happen at the target/current level. The config indirection is just to update the concrete models themselves, given that they are immutable.